cfme/tests/configure/test_access_control.py

```diff
@@ -1097,14 +1099,21 @@ def test_tenant_quota_input_validate(appliance):
 
 
 def test_delete_default_tenant(appliance):
+    """
+    Steps:
+    1. Login as an 'Administrator' user
+    2. Navigate to configuration > access control > tenants
+    3. Select default tenant('My Company') from tenants table
+    4. Delete using 'configuration > Delete selected items'
+    5. Check whether default tenant is deleted or not
+    """
     roottenant = appliance.collections.tenants.get_root_tenant()
-    view = navigate_to(appliance.collections.tenants, 'All')
-    for row in view.table.rows():
-        if row.name.text == roottenant.name:
-            row[0].check()
-    msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
-    with pytest.raises(Exception, match=msg):
-        view.toolbar.configuration.item_select('Delete selected items', handle_alert=True)
+    try:
+        msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
+        tenant = appliance.collections.tenants.instantiate(name=roottenant.name)
+        appliance.collections.tenants.delete(tenant)
+    except NoSuchElementException:
```

Comment:
I think the test needs a bit more here.

We should assert the tenant still exists after the call.

Is there a flash message displayed that we can check? The delete method simply calls `item_select` and returns.

Reply:
@mshriver 
Yes, once we delete the default tenant the flash message displayed i.e. ```'Default Tenant "My Company" can not be deleted'``` which shows default tenant still exist. 
So we can assert this flash message  in **try** block: 
```
view = navigate_to(appliance.collections.tenants, "All")
view.flash.read()[0] == msg
```
Please let me know if you have any other suggestions.
Thank you.

Reply:
@mshriver 
Also, I want to add one more thing:
 - we need to remove exception and just need to check this test case with **assert** message:```view.flash.read()[0] == msg``` only. because exception is already added in **delete()** method.